configs/permissions/vulnerability.json

```diff
@@ -1,3 +1,10 @@
 {
-    "*": ["*"]
+    "*": ["*", "read", "write"],
+    "vulnerability_results": ["read"],
+    "cve.business_risk": ["write"],
+    "cve.status": ["write"],
+    "cve_system.cve.status": ["write"],
+    "report": ["read", "write"],
+    "system": ["exclude", "include"],
+    "remediation": ["create", "update"]
```

Comment:
per our discussion, let's remove this remediation permission

Reply:
Done